xtask/src/commands/release.rs

```diff
@@ -410,3 +414,54 @@ impl Prepare {
         Ok(())
     }
 }
+
+struct PreVerify();
+
+impl PreVerify {
+    fn run() -> Result<()> {
+        // Get the version out of Cargo.toml
+        let metadata = MetadataCommand::new()
+            .manifest_path("./apollo-router/Cargo.toml")
+            .exec()?;
+        let version = format!(
+            "v{}",
+            metadata
+                .root_package()
+                .expect("expected root")
+                .version
+                .to_string()
+        );
+
```

Comment:
Could this be replaced by using `PKG_VERSION`?  As we do in other places?

https://github.com/apollographql/router/blob/2f2462f8b06fda28d21369f7c8e1912efbdb8804/xtask/src/lib.rs#L25-L37

Reply:
[a85f195](https://github.com/apollographql/router/pull/3859/commits/a85f195794e5bd29b50f4d4e46d80a50da770a94)